src/mcp/server/sse.py

```diff
@@ -78,6 +79,18 @@ def __init__(self, endpoint: str) -> None:
         self._read_stream_writers = {}
         logger.debug(f"SseServerTransport initialized with endpoint: {endpoint}")
 
+    async def _remove_stream_writer(self, session_id: UUID) -> None:
+        """
+        Remove the SSE session with the given session ID.
+        """
+        logger.debug(f"Remove SSE session with ID: {session_id}")
+        writer = self._read_stream_writers.pop(session_id, None)
+        if writer:
```

Comment:
Remember the walrus...
```suggestion
        if writer := self._read_stream_writers.pop(session_id, None):
```